nodejs/eks/cluster.ts

```diff
@@ -401,6 +401,7 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
     };
 
     const partition = pulumi.output(aws.getPartition({ provider })).partition;
+    const dnsSuffix = pulumi.output(aws.getPartition({ provider })).dnsSuffix;
```

Comment:
We should be able to fetch both in a single call and grab the two values we need.

```suggestion
    const { partition, dnsSuffix } = pulumi.output(aws.getPartition({ provider }));
```